src/app/application/services/data-source/CallableEventSource.ts

```diff
@@ -25,11 +25,12 @@ export default class CallableEventSource<T = {}> extends EventTarget {
     this._open()
   }
 
-  _open() {
+  protected _open() {
     (async function (self) {
       try {
-        self.readyState = 0
+        self.readyState = CONNECTING
         const source = self.source()
+        self.readyState = OPEN
```

Comment:
There where several things I did whilst checking in DataSource whilst checking this again and following things around/debugging.

This first one was:

1. We have a `CONNECTING=0` constant a little higher up in this file, I guess this was just missed when this was originally written/iterated on
2. I moved setting the readyState to `OPEN` here
    i. to not expose this kind of stuff to the EventSources `Callable`
    ii. use the OPEN constant which I would rather remain private-ish